script.js

```diff
@@ -0,0 +1,71 @@
+let books = [
+  {
+    id: 0,
+    title: 'Lorem ipsum',
+    author: 'Testeroo Testyy',
+  },
+  {
+    id: 1,
+    title: 'Second book',
+    author: 'Testeroo Testyy',
+  },
+
+];
+
+const addBook = (title, author) => {
+  const book = {
+    title,
+    author,
+  };
+
+  books.push(book);
+  return book;
+};
+
+const removeBook = (id) => {
+  books = books.filter((book) => book.id !== id);
+};
+
```

Comment:
- Hey, After deleting the hard-coded books and adding some new books, it doesn't get deleted. please kindly fix this issue.